package.json

```diff
@@ -77,7 +77,7 @@
     "@babel/preset-typescript": "^7.21.5",
     "@babel/runtime": "^7.20.6",
     "@chakra-ui/icons": "^1.1.1",
-    "@chakra-ui/react": "^1.7.4",
```

Comment:
updating package versions should happen in a separate PR, please revert this line change